addons/hr_holidays/static/src/js/dist/time_off_calendar.dev.js

```diff
@@ -0,0 +1,188 @@
+"use strict";
+
+odoo.define('hr_holidays.dashboard.view_custo', function (require) {
+  'use strict';
+
+  var core = require('web.core');
+
+  var CalendarPopover = require('web.CalendarPopover');
+
+  var CalendarController = require("web.CalendarController");
+
+  var CalendarRenderer = require("web.CalendarRenderer");
+
+  var CalendarView = require("web.CalendarView");
+
+  var viewRegistry = require('web.view_registry');
```

Comment:
```suggestion
  var core = require('web.core');
  var CalendarPopover = require('web.CalendarPopover');
  var CalendarController = require("web.CalendarController");
  var CalendarRenderer = require("web.CalendarRenderer");
  var CalendarView = require("web.CalendarView");
  var viewRegistry = require('web.view_registry');
```

Reply:
Wops, this file has been commited but it shouldn't, I'll delete it